logical/system_view.go

```diff
@@ -6,12 +6,12 @@ import "time"
 // for logical backends to consume
 type SystemView interface {
 	// DefaultLeaseTTL returns the default lease TTL set in Vault configuration
-	DefaultLeaseTTL() time.Duration
+	DefaultLeaseTTL() (time.Duration, error)
```

Comment:
Errors happen if the path is invalid. When the lookup function for the path is called, even if we move to using the router, it can return an error. The only way to not expose that error would be to return default duration values (e.g. 0) -- but since that's not actually valid, backends then need to know that they need to check for this condition and do something about it. So it just obfuscates the error condition and potentially leads to incorrect behavior.